README.md

```diff
@@ -1,95 +1,86 @@
-# terraform-equinix-labs
+# Terraform Equinix Labs Project
 
-<!-- TEMPLATE: Review all "TEMPLATE" comments and remove them when applied. -->
-<!-- TEMPLATE: replace "template" with the name of your project. The prefix "terraform-equinix-" informs the Terraform registry that this project is a Terraform module associated with the Equinix provider, preserve this prefix. -->
 [![Experimental](https://img.shields.io/badge/Stability-Experimental-red.svg)](https://github.com/equinix-labs/standards#about-uniform-standards)
-[![run-pre-commit-hooks](https://github.com/equinix-labs/terraform-equinix-labs/actions/workflows/pre-commit.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-template/actions/workflows/pre-commit.yaml)
-[![generate-terraform-docs](https://github.com/equinix-labs/terraform-equinix-labs/actions/workflows/documentation.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-template/actions/workflows/documentation.yaml)
+[![run-pre-commit-hooks](https://github.com/equinix-labs/terraform-equinix-labs/actions/workflows/pre-commit.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-labs/actions/workflows/pre-commit.yaml)
+[![generate-terraform-docs](https://github.com/equinix-labs/terraform-equinix-labs/actions/workflows/documentation.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-labs/actions/workflows/documentation.yaml)
 
-## Usage
-
-This project is experimental and supported by the user community. Equinix does not provide support for this project.
-
-Install Terraform using the [tfenv](https://github.com/tfutils/tfenv) utility.
-
-This project may be forked, cloned, or downloaded and modified as needed as the base in your integrations and deployments.
-
-This project may also be used as a [Terraform module](https://learn.hashicorp.com/collections/terraform/modules).
-
-To use this module in a new project, create a file such as:
-
-```hcl
-# main.tf
-terraform {
-  required_providers {
-    equinix = {
-      source = "equinix/equinix"
-    }
-}
+This repository contains a collection of Terraform modules to help automate the deployment and management of both baremetal and [Kubernetes (EKS-A)](https://github.com/equinix-labs/terraform-equinix-metal-eks-anywhere) environments running on [Equinix Metal](https://deploy.equinix.com/).
```

Comment:
EKS-A is not the focus of this repo.

Reply:
Rather it is one of the many projects that are catalogued in this repo and can be demonstrated through the workshop pattern included in this repo.

Reply:
Resolved in the latest commit. Thx.